knowledge-base/actions/fossa-contrib/fossa-action/action-security.yml

```diff
@@ -0,0 +1,7 @@
+name: Fossa Action
+github-token:
```

Comment:
@sph4674 If there are no permissions present, then there is no need of adding `action-input` or `permissions` field for the kb's instead just mention a comment
```yml
name: Fossa Action
# Only require contents read permissions which gets added at workflow level
```